client/components/payment-activity/index.tsx

```diff
@@ -68,9 +55,16 @@ const PaymentActivity: React.FC = () => {
 
 	const { selectedCurrency } = useSelectedCurrency();
 
+	const [ dateRange, setDateRange ] = useState( {
+		date_start: defaultDateRange.date_start,
+		date_end: defaultDateRange.date_end,
+		key: defaultDateRange.key,
```

Comment:
Suggest using a clearer name for preset name – `key` is too generic.

Reply:
This has been renamed to preset_name. Marking this as resolved.